README.md

```diff
@@ -13,15 +13,15 @@ There are things, in Renuo projects, which are mandatory, other that are suggest
 The basic things that need to be ready before the team can start working on a project are:
 
 * An existing *git* repository containing the project
-* Three main branches: *main*, *develop*
```

Comment:
He nice one. Whoever removed `testing` (or reviewed the change) didn't really read through the result :wink: